data/src/main/java/com/bottlerocketstudios/brarchitecture/data/converter/LinksDtoConverter.kt

```diff
@@ -0,0 +1,10 @@
+package com.bottlerocketstudios.brarchitecture.data.converter
+
+import com.bottlerocketstudios.brarchitecture.data.model.LinkDto
+import com.bottlerocketstudios.brarchitecture.data.model.LinksDto
+import com.bottlerocketstudios.brarchitecture.domain.models.Link
+import com.bottlerocketstudios.brarchitecture.domain.models.Links
+
+fun LinksDto.convertToLinks() = Links(avatar = avatar?.convertToLink())
```

Comment:
@VanyaHuamanBottleRocket Thanks for migrating from unmarked Dto classes to Dto+Domain model classes with converter functions!